src/components/clientDashboard/ProfileLayout.tsx

```diff
@@ -34,7 +35,12 @@ const ProfileLayout: React.FC<Props> = ({ client, notices = [] }) => {
           </Grid>
         )}
         <Grid item md={12} lg={canViewEnrollments ? 6 : 8}>
-          <ClientProfileCard client={client} />
+          <Stack gap={2}>
+            <Box>
+              <ClientProfileCard client={client} />
+            </Box>
```

Comment:
is Box needed?

Reply:
Yeah because the ClientProfileCard was actually a fragment of 2 components 🤔  this was just a quick way to use the `Stack` so I didn't have to touch that other component too much.